### PR TITLE
Add EN HR MessageEase Variant

### PR DESCRIPTION
**Pull request by ssaric**
image
Basically En MessageEase, but with croatian symbols

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ENHRMessagEase.kt [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+                        listOf(
+                                SPACEBAR_KEY_ITEM,
+                                RETURN_KEY_ITEM,
+                        ),
+                ),
+        )
+
+val KB_EN_HR_MESSAGEASE: KeyboardDefinition =
+        KeyboardDefinition(
+                title = "english messagease with croatian guillemets",
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ENHRMessagEase.kt:555]: Look at the other keyboard names. languages first, and in their original characters. Then the type of keyboard last.

**Review by dessalines — COMMENTED** [review]
Also make sure you run `./gradlew formatKotlin` to pass lint.

**Comment by ssaric** [comment]
@dessalines how do I fix formatting locally? I am not a kotlin developer, don't know the setup :smile:

**Comment by dessalines** [comment]
I'll get it.

**Review by dessalines — APPROVED** [review]
Thx!

**Comment by ssaric** [comment]
> Also make sure you run `./gradlew formatKotlin` to pass lint.

missed this. Sorry. Thank you for this lifesaver!

**Comment by dessalines** [comment]
No probs!

**Pull request merged**